Resources/Prototypes/Loadouts/Jobs/Civilian/passenger.yml

```diff
@@ -57,6 +57,15 @@
   equipment:
     jumpsuit: ClothingUniformJumpsuitAncient
 
+# The Next Generation
```

Comment:
this should be in our file structure and not editing an existing file! make a Resources/Prototypes/_Impstation/Loadouts/Jobs/Civilian/passenger.yml and move this entity there